src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -122,7 +122,7 @@ def ReissnerNordstorm(
     sch=symbols("r_s"),
     Q=symbols("Q"),
 ):
-    """"
+    """ "
```

Comment:
What's this?

Reply:
@shreyasbapat The extra quotation mark? Seems like, it was mistyped. Removing it.

Reply:
There was an `isort` update, which broke the quality checks (https://app.circleci.com/pipelines/github/einsteinpy/einsteinpy/1885/workflows/130a38b8-2db8-4b63-a0a1-fc0cf7fdeadd/jobs/8730). I guess, while making the changes then, I mistyped this. And, `isort` update is also why, there are many extraneous edits in this PR.

Reply:
Oh, well. I have some edits queued in the `warp` branch, that adds the Alcubierre Metric. I will push this change there (#533).